docs/hosting-a-server-with-northstar/prerequisites.md

```diff
@@ -1,6 +1,6 @@
 # Prerequisites
 
-**TL;DR:** Port forward `37015` (UDP) and `8081` (TCP)
```

Comment:
if we drop the "TL;DR" we should just remove the whole line. Otherwise it's not clear to someone not experienced with networking why this part is there.

Reply:
I added it back as a short version